tekton/resources/nightly-tests/base/deploy_tekton_component.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: deploy-tekton-component-nightly
+spec:
+  params:
+  - name: package
+    description: package to install
+  - name: version
+    description: version to install
+    default: latest
+  - name: extra-file
+    description: extra file to install (for instance, interceptors.yaml for triggers)
+    default: ""
+  workspaces:
+  - name: k8s-shared
+    description: workspace for k8s config, configuration file is expected to have `config` name
+    mountPath: /root/.kube
+  steps:
+  - name: deploy
+    image: gcr.io/tekton-releases/dogfooding/kubectl:latest
+    env:
+    - name: KUBECONFIG
+      value: $(workspaces.k8s-shared.path)/config
+    command:
+    - /bin/sh
+    args:
+    - -ce
+    - |
+      kubectl apply --filename https://storage.googleapis.com/tekton-releases-nightly/$(params.package)/$(params.version)/release.yaml
+      if [ "$(params.extra-file)" != "" ]; then
```

Comment:
I think you'll need to wait for CRD to be established before you install the extra manifest - see https://github.com/tektoncd/plumbing/blob/1ffa936d254ffba2565a248371fbd50ce8f29987/tekton/resources/release/base/install_tekton_release.yaml#L105